src/js/index.js

```diff
@@ -16,17 +16,16 @@ model.init();
 var ui = require('./components');
 
 var g = model.Scene.child('marks.group'),
-    g2 = g.child('marks.group'),
-    g3 = g2.child('marks.group'),
+    g2 = model.Scene.child('marks.group'),
     p = model.pipeline('cars'),
     p2 = model.pipeline('jobs'),
     p3 = model.pipeline('gapminder');
 
 // Pre-populate state with one rect, one symbol, one text & one line mark
-g3.child('marks.rect');
-g.child('marks.symbol');
+g2.child('marks.rect');
+g2.child('marks.symbol');
 g2.child('marks.line');
-g.child('marks.text');
+g2.child('marks.text');
```

Comment:
This is one of those "discussed in slack" things that I will make sure we capture in a git issue.  For our deadline, we are just going to have the non nested groups in the UI. I will open an issue about designing the visuals for nested groups.